lib/config.sh

```diff
@@ -1,12 +1,19 @@
 #! /bin/bash
```

Comment:
This can live in _config/_, especially if we expect other files to live here. Or, it could live in repo root?

Reply:
What do you think about dividing it into separate files (GCE, AWS, and Common).   Some shared values (number of instances, naming, etc) don't need to be defined twice, while instance type, image ids, etc, are unique to providers. 

Each config could live in the same dir as it's provider's utility functions, or in a common configs/ dir.

Reply:
I think that's a good way to start out and let it evolve with usage.